internal/ffmpeg/exec.go

```diff
@@ -112,7 +124,7 @@ func (cmd *TranscodeCmd) OutputPath() string {
 }
 
 func (cmd *TranscodeCmd) String() string {
-	var pid int = -1
+	pid := -1
```

Comment:
Prefer `:=` in most cases?

I do like it.

Reply:
Haha yeah, cleaner than an if-else block to initialise it IMO :)